stacker/actions/build.py

```diff
@@ -271,26 +271,10 @@ def _handle_missing_parameters(self, params, required_params,
         return params.items()
 
     def _generate_plan(self, tail=False):
-        plan_kwargs = {}
-        if tail:
-            plan_kwargs["watch_func"] = self.provider.tail_stack
```

Comment:
Yeah - this is used by some people with `--tail`.  Does it cause issues if you leave it in?